blockwise/blockwise.go

```diff
@@ -0,0 +1,721 @@
+package blockwise
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"sync"
+	"time"
+
+	"github.com/dsnet/golib/memfile"
+	"github.com/go-ocf/go-coap/v2/message"
+	"github.com/go-ocf/go-coap/v2/message/codes"
+	"github.com/patrickmn/go-cache"
+)
+
+const (
+	maxBlockNumber = int(1048575)
+)
+
+// SZX enum representation for szx
+type SZX uint8
```

Comment:
I'm not sure if this name **SZX** is correct

Reply:
Updated doc about SZX type.